### DISCUSSION TITLE
Followers count not showing properly

### DISCUSSION THREAD

**Discussion by stabldev** [community/community#142299] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

My account got suspended 2 days ago, just got it back. But when I check profile > followers count, its showing 0, but there are followers if I go to [followers tab](https://github.com/moonlitgrace?tab=followers). I replied mentioning this in the mail I got, but dropping this here as well.

**Comment by stabldev** [comment]
seems fixed now.

**Reply by ayurie2627-sudo** [reply]
> seems fixed now.
